geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/CacheRealizationFunction.java

```diff
@@ -77,14 +79,23 @@ public void execute(FunctionContext<List> context) {
     }
   }
 
-  public Object executeGet(InternalCache cache,
+  public RuntimeInfo executeGet(FunctionContext<List> context,
+      InternalCache cache,
       CacheElement cacheElement) {
     ConfigurationRealizer realizer = realizers.get(cacheElement.getClass());
 
     if (realizer == null) {
       return null;
     }
-    return realizer.get(cacheElement, cache);
+    RuntimeInfo runtimeInfo = realizer.get(cacheElement, cache);
+
+    // set the membername if this is not a global runtime
+    if (cacheElement instanceof CorrespondWith
```

Comment:
The comment here is help to the future reader of this class. Perhaps in lieu of the comment and the long boolean expression, you could replace it with a predicate function `isGlobalRuntime(CacheElement cacheElement)`. This makes it easier to read and prevents that comment from rotting (since we can remove it).

Reply:
normally I would put it in a predicate function if it's been used multiple times, but in this case, it's only used once, and (x instanceof y) is a more visually clear than a function no matter how descriptive the name is.